eZ/Publish/Core/MVC/Symfony/Controller/Content/DownloadController.php

```diff
@@ -6,6 +6,7 @@
 
 use eZ\Bundle\EzPublishIOBundle\BinaryStreamResponse;
 use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Values\Content\Content;
 use eZ\Publish\API\Repository\Values\Content\Field;
 use eZ\Publish\Core\Helper\TranslationHelper;
```

Comment:
guess it can be removed from here now as it is done by ContentDownloadRouteReferenceListener right?